woocommerce/woocommerce-ios
# [My store - Analytics] Add new enum case to handle custom date range

Part of: #11973 


## Description

Adds new enum cases to support the custom date range selection. 
This PR just adds the new case to unblock development for UI and API. We have added TODOs in order to update the code accordingly in future UI or API related PRs.

## Testing instructions
We need to test that analytics works as before.
1. Login into the app
2. In the "My Store" tab, ensure that the analytics section works data as before.
3. Switch between "Today", "This Week", "This Month", This Year" and tabs and UI works as before.
4. Tap on "See more", and the analytics detail screen should work as before.


## Screenshots

![Simulator Screen Recording - iPhone 15 Pro Max - 2024-02-13 at 07 55 47](https://github.com/woocommerce/woocommerce-ios/assets/524475/5b2f5df3-05f7-4b5d-a6fe-49051a3309a5)


---
